src/chrono.rs

```diff
@@ -0,0 +1,65 @@
+//! chrono types
+//!
+//! based on Dart VM, microseconds unit is used
+
+use crate::{ffi::DartCObject, IntoDart};
+
+impl IntoDart for chrono::DateTime<chrono::Utc> {
+    /// on the other side of FFI, value should be reconstructed like:
+    ///
+    /// - hydrate into Dart [DateTime](https://api.flutter.dev/flutter/dart-core/DateTime/DateTime.fromMicrosecondsSinceEpoch.html)
+    ///   ```dart,ignore
+    ///   DateTime.fromMicrosecondsSinceEpoch(raw, isUtc: true);
+    ///   ```
+    ///   > beware that conversion is lossy, see more details in [Dart SDK issue](https://github.com/dart-lang/sdk/issues/44876).
+    ///
+    /// - hydrate into Rust [DateTime](chrono::DateTime)::<[Utc](chrono::Utc)>
+    ///   ```rust,ignore
+    ///   let s = (raw / 1_000_000) as i64;
+    ///   let ns = (raw.rem_euclid(1_000_000) * 1_000) as u32;
+    ///   chrono::DateTime::<chrono::Utc>::from_utc(chrono::NaiveDateTime::from_timestamp(s, ns), chrono::Utc);
+    ///   ```
+    /// 
+    ///   note that it could overflow under the same conditions as of [chrono::NaiveDateTime::from_timestamp](https://docs.rs/chrono/latest/chrono/naive/struct.NaiveDateTime.html#method.from_timestamp)
```

Comment:
```suggestion
    ///   note that it could overflow under the same conditions as of [chrono::NaiveDateTime::from_timestamp](https://docs.rs/chrono/0.4.22/chrono/naive/struct.NaiveDateTime.html#method.from_timestamp)
```

Reply:
ok in c3cd8dd35944fba13586382d7be26679cd533980